llvm/unittests/Analysis/VectorFunctionABITest.cpp

```diff
@@ -14,24 +15,31 @@
 using namespace llvm;
 
 namespace {
-// Test fixture needed that holds the veariables needed by the parser.
+/// Perform tests against VFABI Rules. `invokeParser` creates a VFInfo object
+/// and a scalar FunctionType, which are used by tests to check that:
+/// 1. The scalar and vector names are correct.
+/// 2. The parameter number from the parsed mangled name matches the scalar ones
```

Comment:
nit: "The number of parameters from the parsed mangled name matches the number of arguments in the scalar function passed as FunctionType string."